### PR TITLE
Added -Location parameter to Set- AzVMRunCommand

### PR DESCRIPTION
**Pull request by pselugar**
## Description

- Updated Set -AzVMRunCommand  example to have Location as a parameter
- Made some typo corrections to the cmdlet output 
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative

- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)

### CODE REVIEW

File: src/Compute/Compute.Autorest/examples/Set-AzVMRunCommand.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ### Example 1: Simple Example
 ```powershell
-PS C:\> Set-AzVMRunCommand -ResourceGroupName $rgname -VMName $vmname -RunCommandName 'firstruncommand' 
+PS C:\> Set-AzVMRunCommand -ResourceGroupName $rgname -VMName $vmname -Location $locationname -RunCommandName 'firstruncommand'
```
Comment (BethanyZhou) [inline src/Compute/Compute.Autorest/examples/Set-AzVMRunCommand.md:3]:
```suggestion
Set-AzVMRunCommand -ResourceGroupName $rgname -VMName $vmname -Location $locationname -RunCommandName 'firstruncommand' 
```

**Review by BethanyZhou — APPROVED** [review]
Looks good to me.

File: src/Compute/Compute.Autorest/examples/Set-AzVMRunCommand.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 ### Example 1: Simple Example
 ```powershell
-PS C:\> Set-AzVMRunCommand -ResourceGroupName $rgname -VMName $vmname -RunCommandName 'firstruncommand' 
+Set-AzVMRunCommand -ResourceGroupName $rgname -VMName $vmname -Location $location -RunCommandName 'firstruncommand'
```
Comment (grizzlytheodore) [inline src/Compute/Compute.Autorest/examples/Set-AzVMRunCommand.md:3]: lets make it -Location "eastus" to match the output below

Comment (grizzlytheodore) [reply src/Compute/Compute.Autorest/examples/Set-AzVMRunCommand.md:3]: and PS C:\>

Comment (pselugar) [reply src/Compute/Compute.Autorest/examples/Set-AzVMRunCommand.md]: Fixed, good catch!

**Review by BethanyZhou — APPROVED** [review]

**Pull request merged**